UKHomeOffice/nmsw-ui
# NMSW-521 Remove whitespace from FAL 5 template

# Ticket

NMSW-521

----

## AC

We have found that the FAL 5 template contains whitespace on row 7 which is causing the fields to error incorrectly.
We need to remove the whitespace from the template and replace the current FE template with the new FAL 6 template (without white space)

Current behaviour: 
- You download a fresh Crew details template
- You upload the empty template to Crew details upload
- You are taken to the errors found page with errors relating to row 7 fields required

Expected behaviour: 
- You download a fresh Crew details template
- You upload the empty template to Crew details upload
- You you get an error in the error summary telling you that 'Template is empty'

----

## To test

- Run app
- Sign in
- Go to templates tab
- Download a fresh Crew details FAL 5 template
- Go to Your voyages
- Start a voyage report
- Get to Crew details FAL 5 upload
- Upload the blank file you just downloaded
- You should get a 'Template is empty' error

----

## Notes
